doc/classes/VisualServer.xml

```diff
@@ -1105,6 +1130,7 @@
 			<argument index="5" name="quality" type="int" enum="VisualServer.EnvironmentDOFBlurQuality">
 			</argument>
 			<description>
+				Sets the values to be used with the "dof near blur" post-process effect. See [Environment] for more details.
```

Comment:
It appears in the inspector as "Dof Near Blur", so I think we should probably capitalize it similarly here (same for "adjustments" above). Might also remove the need for quotes.

Reply:
Alternatively, keep it to natural language and capitalize DoF as it should be.